filter/smart_feed.go

```diff
@@ -62,13 +74,7 @@ func (feed *Feed) Project(projectlabelname, projectname, interesting_review_stri
 	}
 	feed.AddEntry(project_message_entry)
 
-	project_reviews_messages := fmt.Sprintf("list:(<%[1]s-reviews>) OR to:%[1]s+reviews", projectname)
-
-	project_review_entry := NewEntry("project_review" + strconv.Itoa(len(feed.Entries)))
-	project_review_entry.AddProperty("hasTheWord", project_reviews_messages)
-	project_review_entry.AddProperty("shouldArchive", "true")
-
-	feed.AddEntry(project_review_entry)
+	project_reviews_messages := fmt.Sprintf("list:(<%[1]s-reviews>) OR to:(%[1]s-team+reviews OR %[1]s-sre+reviews OR %[1]s-eng+reviews)", projectname)
 
 	if interesting_review_string != "" {
```

Comment:
Could you check project_reviews_message in Project()? Must it be the same as in Review()?
I've decided to leave it in Project because it is interesting for project. If I move it in Review, it will be 4 parameters. It is a lot.